master/getting-started/docker/tutorials/advanced-label-policy.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Advanced Container Labels Policy
+---
+
+## Background
+
+With Calico as a Docker network plugin, Calico can be configured to extract the
+labels on a container and apply them to the workload endpoint for use with
+Calico
+[policy]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/policy).
+When Calico is configured to use Container Labels, profiles are not created and 
+have no impact on any container traffic.  Because of this, all traffic must be
+explicitly allowed through configuration of the globally defined policy which
+uses selectors to determine which subset of the policy is applied to each
+container based on their labels.  This approach provides a powerful way to
+group together all of your network Policy, makes it easy to reuse policy in
+different networks, and makes it easier to define policy that extends across
+different orchestration systems that use Calico.
+
+## Enabling Docker Networking Container Labels Policy
+
+To enable the Container Labels Policy with Docker Networking the flag
```

Comment:
Perhaps should spell out that all calico/node containers need to be started with this option to avoid a label/profile mix